packages/web-runtime/tests/unit/components/Topbar/FeedbackLink.spec.ts

```diff
@@ -1,21 +1,19 @@
 import FeedbackLink from '../../../../src/components/Topbar/FeedbackLink.vue'
-import { axe, toHaveNoViolations } from 'jest-axe'
 import { defaultPlugins, mount } from 'web-test-helpers'
 
-expect.extend(toHaveNoViolations)
+// expect.extend(toHaveNoViolations)
 
 describe('FeedbackLink component', () => {
   // FIXME: skip flaky test, also see: https://github.com/owncloud/web/issues/8244
-  // eslint-disable-next-line jest/no-disabled-tests
   it.skip('has no accessibility violations', async () => {
     const { wrapper } = getWrapper()
-    expect(
-      await axe(wrapper.html(), {
-        rules: {
-          region: { enabled: false }
-        }
-      })
-    ).toHaveNoViolations()
+    // expect(
+    //   await axe(wrapper.html(), {
+    //     rules: {
+    //       region: { enabled: false }
+    //     }
+    //   })
+    // ).toHaveNoViolations()
```

Comment:
There is a comment already, this test has always been flaky and I don't know why and how to fix it 🤷

Reply:
Oh I thought we don't use `axe` anymore... if I remember correctly this was just a quick test / use case for automated accessibility tests. Since we didn't follow up I'd say completely get rid of it for now (here and in `OcTable`).